### PR TITLE
Making offset reset, file age, and partition finalization delay configurable

### PR DESCRIPTION
**Pull request by nmurthy**
Increase back to default of 1 hour

### CODE REVIEW

File: src/main/config/defaults.properties [hunk 1]
```diff
@@ -405,6 +405,6 @@ secor.file.reader.writer.factory=com.pinterest.secor.io.impl.ProtobufParquetFile
 secor.max.file.size.bytes=209715200
 # 1 hour
 # for hourly ingestion/finalization, set this property to smaller value, e.g. 1800
-secor.max.file.age.seconds=60
+secor.max.file.age.seconds=1800
```
Comment (efeller) [inline src/main/config/defaults.properties]: Isn't one hour 3600 seconds and not 1800?

**Comment by efeller** [comment]
Can you also copy over latest protobuf from platypus protocols as this PR? Thanks.

File: src/main/config/defaults.properties (hunk 1 above)
Comment (nmurthy) [reply src/main/config/defaults.properties]: yeah but every 30 minutes is a safer value that'd still be beneficial. we can go 3600 though if it works well.

**Comment by nmurthy** [comment]
@efeller I like having PRs reflect one conceptual change at a time, so I'm going to open a separate one to copy the protobufs in

**Comment by nmurthy** [comment]
@efeller bump on merging this

File: src/main/config/defaults.properties [hunk 2]
```diff
@@ -155,7 +155,7 @@ kafka.zookeeper.path=/
 # Only used if kafka.offsets.storage is set to "kafka"
 # http://kafka.apache.org/documentation.html#oldconsumerconfigs
 # Possible values: true or false
-kafka.dual.commit.enabled=true
+kafka.dual.commit.enabled=false
```
Comment (efeller) [inline src/main/config/defaults.properties]: Can you please explain why was this turned off?

File: src/main/config/defaults.properties [hunk 3]
```diff
@@ -106,8 +106,8 @@ secor.gs.path=
 secor.gs.credentials.path=
 
 # Zookeeper config.
-zookeeper.session.timeout.ms=3000
-zookeeper.sync.time.ms=200
+zookeeper.session.timeout.ms=5000
+zookeeper.sync.time.ms=2000
```
Comment (efeller) [inline src/main/config/defaults.properties]: Why did you change these times? What was the reason?

File: src/main/config/defaults.properties [hunk 4]
```diff
@@ -192,7 +192,7 @@ secor.messages.per.second=1000000
 secor.offsets.per.partition=10000000
 secor.offsets.prefix=offset=
 ## How long does it take for secor to forget a topic partition. Applies to stats generation only.
-secor.topic_partition.forget.seconds=600
+secor.topic_partition.forget.seconds=-1
```
Comment (efeller) [inline src/main/config/defaults.properties]: Please explain why was this set to -1 and what is the impact of this setting.

File: src/main/config/defaults.properties [hunk 5]
```diff
@@ -353,7 +353,7 @@ secor.s3.alter.path.date=
 secor.s3.alternative.path=
 
 # If enabled, add calls will be made to qubole, otherwise, skip qubole call for finalization
-secor.enable.qubole=true
+secor.enable.qubole=false
```
Comment (efeller) [inline src/main/config/defaults.properties]: Why was this disabled? What is the impact?

File: src/main/config/defaults.properties [hunk 6]
```diff
@@ -403,7 +403,8 @@ secor.file.reader.writer.factory=com.pinterest.secor.io.impl.ProtobufParquetFile
 # Upload policies.
 # 200 MB
 secor.max.file.size.bytes=209715200
-# 60 seconds
-secor.max.file.age.seconds=60
+# 30 mins
+# finalize files every 1 hour
```
Comment (efeller) [inline src/main/config/defaults.properties]: Not every hour, every 30 minutes. Please adjust comment.
